third_party/BUILD

```diff
@@ -163,8 +163,10 @@ distrib_java_import(
     jars = [
         "api_client/google-api-client-1.22.0.jar",
         "api_client/google-api-client-jackson2-1.22.0.jar",
```

Comment:
I think we can remove two jackson2 jars in this PR.

Reply:
wouldn't that require us to update test code `import com.google.api.client.json.jackson2.JacksonFactory;` as well?

Reply:
Right. I thought only `:jackson2` is required for the test code. Please add them back and we are ready to merge!

Reply:
pr updated